tests/test_asgi.py

```diff
@@ -23,6 +23,15 @@ async def echo_path(scope, receive, send):
     await send({"type": "http.response.body", "body": output})
 
 
+async def echo_raw_path(scope, receive, send):
+    status = 200
+    output = json.dumps({"raw_path": repr(scope["raw_path"])}).encode("utf-8")
```

Comment:
I'm using `repr(scope["raw_path"])` here as an easy way to confirm that the value is indeed a Python bytestring - it ends up being returned as `"b'/user%40example.org'"` in the JSON.

Reply:
Yup that make sense.

I guess it might look a bit confusing to future readers, so we could either:

1. Comment it.
2. Switch it around to `{"raw_path": scope["raw_path"].decode("ascii")}`

I'd marginally prefer (2), since it's actually perfectly sufficient here, but I'm okay enough with either.